modules/05_add_blob_storage/README.md

```diff
@@ -0,0 +1,127 @@
+# Add persistent storage for images
+
+## Goal
+
+Modify the app to add persistent storage for images.
+
+## Tutorial
+
+> In this tutorial we will assume that you've completed the previous tutorial
+and already have a game where you get an image and you have to guess a word that describes it.
+Now, we'll need to change the image storing functionality and upload the images to the cloud.
+Then images could be served from the cloud.
+
+### Part 1. Create the storage account
+
+#### Intro
+
+[Azure Blob Storage](https://docs.microsoft.com/azure/storage/blobs/) is Microsoft's object storage solution
+ for the cloud. We will use this to store our image and also to get them from there using an URL. It's advisable that
+ you read about this service and familiarise yourself with the concepts of a storage account, container and blob.
+
+#### 1. Create a storage account in your own subscription where you will save the images
+
+In order to store the images, we recommend that you create your own storage account using the portal.
+The required steps are provided [here](https://docs.microsoft.com/azure/storage/common/storage-account-create?tabs=azure-portal)
+
+#### 2. Save and expose the storage account information
+
+In order to connect to the storage account from our code, we will need the name of the storage account and a
+ storage account key that can be found in the Access Keys of the storage account like shown
+ [here](https://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-python#copy-your-credentials-from-the-azure-portal).
+Don't copy the connection string, copy the key instead, as we will use this to build the connection string
+ and also to get the URL for each uploaded file.
+These should be added to two environment variables: AZURE_STORAGE_ACCOUNT_NAME and AZURE_STORAGE_ACCOUNT_KEY.
+An example on how to add an environment variable is [here](https://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-python#configure-your-storage-connection-string).
```

Comment:
I think this particular link isn't helpful. They'll rather need an instruction how to set env variables in vscode .